From inaky@linux.intel.com  Sat Dec 20 21:10:26 2008
From: Inaky Perez-Gonzalez <inaky@linux.intel.com>
Date: Sat, 20 Dec 2008 16:57:59 -0800
Subject: wimax: export linux/wimax.h and linux/wimax/i2400m.h with headers_install
To: netdev@vger.kernel.org
Cc: wimax@linuxwimax.org, greg@kroah.com, David Woodhouse <dwmw2@infradead.org>
Message-ID: <2a7ebc201ca59bbac909e77248eddc1c39fd1a93.1229820957.git.inaky@linux.intel.com>

From: Inaky Perez-Gonzalez <inaky@linux.intel.com>

These two files are what user space can use to establish communication
with the WiMAX kernel API and to speak the Intel 2400m Wireless WiMAX
connection's control protocol.

Signed-off-by: Inaky Perez-Gonzalez <inaky@linux.intel.com>
Cc: David Woodhouse <dwmw2@infradead.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/Kbuild       |    2 ++
 include/linux/wimax/Kbuild |    1 +
 2 files changed, 3 insertions(+)

--- a/include/linux/Kbuild
+++ b/include/linux/Kbuild
@@ -376,3 +376,5 @@ unifdef-y += xattr.h
 unifdef-y += xfrm.h
 
 objhdr-y += version.h
+header-y += wimax.h
+header-y += wimax/
--- /dev/null
+++ b/include/linux/wimax/Kbuild
@@ -0,0 +1 @@
+header-y += i2400m.h
